src/main/resources/migrations/00035_index_columns_for_usage_report.sql

```diff
@@ -0,0 +1,9 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:index_type_products
+CREATE INDEX products_type_idx ON products(type);
+-- rollback drop index products_type_idx
+
+--changeset uk.gov.pay:index_date_created_payments
+CREATE INDEX payments_date_created_idx ON payments(date_created);
```

Comment:
If query uses max(date_created) group by product_id, we also need to add product_id to the index.